web/wdio.conf.ts

```diff
@@ -309,8 +329,15 @@ export const config: WebdriverIO.Config = {
      * @param {boolean} result.passed    true if test has passed, otherwise false
      * @param {object}  result.retries   information about spec related retries, e.g. `{ attempts: 0, limit: 0 }`
      */
-    // afterTest: function(test, context, { error, result, duration, passed, retries }) {
-    // },
+    afterTest: async function (
```

Comment:
Allow testers to observe the end result of the test, if so desired, to verify the CSS "looks okay."